tokens/test/erc-721-minimal/RaribleUser.test.js

```diff
@@ -36,6 +42,34 @@ contract("ERC721RaribleUser minimal", accounts => {
   	assert.equal(await token.supportsInterface("0x780e9d63"), true);
   });
 
+  it("check for support IERC2981 interface", async () => {
+  	assert.equal(await token.supportsInterface("0x2a55205a"), true);
+  });
+
+  it("check Royalties IERC2981", async () => {
+    testRoyaltyV2981Calculate = await TestRoyaltyV2981Calculate.new();
```

Comment:
variable without initialisation

Reply:
testRoyaltyV2981Calculate  - object of test contract TestRoyaltyV2981Calculate
need only for call Library method
LibRoyalties2981.calculateRoyalties
i suppose we don`t need any initialisation

Reply:
i mean it's better to always initialize varibles with let or const or it could lead to problems sometimes
making 50th line:
let testRoyaltyV2981Calculate = await TestRoyaltyV2981Calculate.new(); 
is enough

Reply:
Oh, sorry, I don`t understand, I agree